.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "@babel/eslint-parser",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb", "plugin:react/recommended", "plugin:react-hooks/recommended"],
+  "plugins": ["react"],
+  "rules": {
+    "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
+    "react/react-in-jsx-scope": "off",
+    "import/no-unresolved": "off",
+    "no-shadow": "off"
+  },
+  "ignorePatterns": [
+    "dist/",
```

Comment:
- [Optional ] Please it would be better to delete all React unnecessary files like App.CSS, App. test, and all folders that you didn't use as it is better to put just the files that you are actually going to use.
- [ Optional ] Please try to use just one App.js file as it is confusing. 
![image](https://user-images.githubusercontent.com/74744787/211852196-b21d963e-5c7c-41c5-b401-c5264bcacea6.png)